sourcetools/com.ibm.jpp.preprocessor/com/ibm/jpp/om/Builder.java

```diff
@@ -375,6 +378,23 @@ public boolean build() {
 						getLogger().log("Exception occured in file " + sourceFile.getAbsolutePath() + ", preprocess failed.", 3, t);
 						handleBuildException(t);
 					} finally {
+                                                if (outputFile.exists()) {
+							try {
+								// Load FileAttribute.Tag class and grab constructor
```

Comment:
Please correct indentation and finish each sentence with a period.
This is not reasonable to apply to platforms other than z/OS (where the reflection error is printed nearly 2000 times).